cmd/brimcap/info/command.go

```diff
@@ -16,7 +16,7 @@ import (
 
 var Info = &charm.Spec{
 	Name:  "info",
-	Usage: "info <input_pcap>",
+	Usage: "info pcap",
```

Comment:
Isn't the original text with the `< >` the appropriate convention since the input pcap filename is a required arg?

Reply:
You are probably right, I'm not sure the convention for this in command arguments.